tests/common.go

```diff
@@ -4064,12 +4064,41 @@ func ParseFlags() {
 			}
 		}
 
+		/*
+			Get TestSetID based on below precedence
+			1. Check if user has passed in command line and use
+			2. Check if user has set it has an env variable and use
+			3. Check if build.properties available with TestSetID
+			4. If not present create a new one
+		*/
 		val, ok := os.LookupEnv("DASH_UID")
-		if ok {
+		if testsetID != 0 {
+			dash.TestSetID = testsetID
+		} else if ok && (val != "" && val != "0") {
 			testsetID, err = strconv.Atoi(val)
 			if err != nil {
 				log.Warnf("Failed to convert environment testset id  %v to int, err: %v", val, err)
 			}
+			log.Infof(fmt.Sprintf("Using TestSetID: %s set as enviornment variable", val))
+		} else {
+			fileName := "/build.properties"
```

Comment:
how are we copying build.properties inside torpedo contanier?

Reply:
This has to be mounted with the docker command